python/grass/pygrass/modules/tests/grass_pygrass_grid_test.py

```diff
@@ -127,16 +127,22 @@ def run_grid_module():
         assert info["min"] > 0
 
 
-@pytest.mark.parametrize("clean", [True, False])
-def test_cleans(tmp_path, clean):
+@pytest.mark.parametrize(
+    "clean, surface",
+    [
+        (True, False),
+        ("surface", "non_exist_surface"),
+    ],
+)
+def test_cleans(tmp_path, clean, surface):
```

Comment:
Is you intention to test all combinations? I think this is done with ([pytest-mark-parametrize](https://docs.pytest.org/en/6.2.x/parametrize.html#pytest-mark-parametrize-parametrizing-test-functions)):

```python
@pytest.mark.parametrize("x", [0, 1])
@pytest.mark.parametrize("y", [2, 3])
def test_foo(x, y):
    pass
```

Reply:
I use same pytest-mark-parametrize syntax as use `test_tilling` test function. https://github.com/OSGeo/grass/blob/8abf34e2143f2f9babbeac7b1a62db0393c09ae0/python/grass/pygrass/modules/tests/grass_pygrass_grid_test.py#L222-L230

Reply:
The syntax itself is okay. Each says something different. One `mark.parametrize` has a list of all combinations for the given parameters while multiple mark.parametrize are combined into n x m matrix.

Reply:
Yes you right, I understand now.